react_ws_src/src/helpers/__tests__/computer_moves-test.js

```diff
@@ -0,0 +1,114 @@
+jest.unmock('../computer_moves');
```

Comment:
Im not sure why I had to unmock the file? And I couldn't find any documentation on `jest v13`. It seemed to work, so I left it.